packages/react/src/components/form/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,145 @@
+import type { ReactNode, TextareaHTMLAttributes } from 'react';
+import React, { useState, useId, forwardRef } from 'react';
```

Comment:
Maybe remove this that Github actions points out?

Reply:
Fixed!